.gitignore

```diff
@@ -260,6 +260,7 @@ pip-log.txt
 /chpl/deploy-webapp.sh
 /chpl/.recommenders/
 *-api.sh
+push-*.sh
```

Comment:
These files aren't needed any more, so shouldn't be in the gitignore thing. All of the push scripts were merged into the single push.sh one